pigeon.py

```diff
@@ -176,32 +189,15 @@ def is_crash_id(crash_id):
 
     """
     return (
-        # Verify length of the string
-        len(crash_id) == 36  # and
+        # Match structure
+        bool(CRASH_ID_RE.match(crash_id)) and
 
         # The 7-to-last character is a throttle result
-        # FIXME(willkg): We can re-enable this check later after
-        # -prod has been updated to use Antenna.
-        # crash_id[-7] in (ACCEPT, DEFER)
+        crash_id[-7] in (ACCEPT, DEFER)
     )
 
 
-def extract_bucket_name(record):
-    """Given a record, extracts the bucket name
-
-    :arg dict record: the AWS event record
-
-    :returns: None (no clue what happened) or the bucket name
-
-    """
-    try:
-        return record['s3']['bucket']['name']
-    except KeyError:
-        logger.exception('Exception thrown when extracting bucket name.')
-        return None
```

Comment:
I dropped this since all S3 events we're getting will have this key. I fixed the tests to include it in our faux-event.